lib/webrtc/getstats.js

```diff
@@ -329,9 +605,21 @@ function chromeOrSafariGetTrackStats(peerConnection, track, options) {
       }, null, reject);
     });
   }
-  return peerConnection.getStats(track).then(response => {
-    return standardizeChromeOrSafariStats(response, options);
-  });
+
+  return peerConnection.getStats(track)
+    .then(response => {
+      return standardizeChromeOrSafariStats(response, options);
+    })
+    .catch(() => {
+      // NOTE(lrivas): Citrix doesn't support track-specific getStats,
+      // so this workaround tries getting all stats and filtering by track.
+      return peerConnection.getStats()
+        .then(stats => {
+          const isRemote = isRemoteTrack(peerConnection, track);
+          const filteredStats = filterStatsByTrack(stats, track, isRemote);
```

Comment:
Add an info level log before and after calling filterStatsByTrack.

Reply:
Done!